woff2/index.html

```diff
@@ -913,6 +971,7 @@ <h3 id="woff20Header-0"><span class='secno'>3.2. </span>WOFF2 Header</h3>
       <tr><td>UInt8</td><td>bboxBitmap[]</td><td>Bitmap (a numGlyphs-long bit array) indicating explicit bounding boxes</td></tr>
       <tr><td>Int16</td><td>bboxStream[]</td><td>Stream of Int16 values representing glyph bounding box data</td></tr>
       <tr><td>UInt8</td><td>instructionStream[]</td><td>Stream of UInt8 values representing a set of instructions for each corresponding glyph</td></tr>
+      <tr><td><del cite="#c2"></del><ins cite="#c2">UInt8</ins></td><td><del cite="#c2"></del><ins cite="#c2">overlapFlagStream[]</ins></td><td><del cite="#c2"></del><ins cite="#c2">Stream of numGlyphs bit values representing overlap flags for each glyph.</ins></td></tr>
```

Comment:
I'd suggest renaming this stream to "overlapSimpleBitmap[]", which is what we agreed to as part of the [WG email discussion](https://lists.w3.org/Archives/Public/public-webfonts-wg/2021May/0013.html) reusing the naming conventions with other similar structures (bboxBitmap[]). 
In any case, even if we decide to adopt a different name like overlapSimpleFlagStream[] - I want to make sure that it clearly indicates that it contains the data for simple glyphs only. Composite glyphs have their own set of flags defined (including the overlap flag), and those are encoded as part of the compositeStream[].

Reply:
Done